rmf_traffic/src/rmf_traffic/geometry/Box.cpp

```diff
@@ -98,17 +98,23 @@ double Box::get_y_length() const
 //==============================================================================
 FinalShape Box::finalize() const
 {
+  double characteristic_length = 0.5 * std::sqrt(
```

Comment:
I modified the characteristic length for Box to be equal to half the diagonal. Would that be more appropriate?

Reply:
Ah yeah, I think that's a more accurate characteristic length.